src/csharp/Microsoft.Spark/ML/Feature/HashingTF.cs

```diff
@@ -75,7 +76,7 @@ public HashingTF Save(string path)
         /// <summary>
         /// Gets the binary toggle that controls term frequency counts
         /// </summary>
-        /// <returns>bool</returns>
+        /// <returns>bool showing term frequency counts</returns>
```

Comment:
it doesn't "show" right? I think it "controls" as the summary says. `Flag that controls term frequency counts`?